### DISCUSSION TITLE
get in contact

### DISCUSSION THREAD

**Discussion by NobodzzZ** [garlett/multiseat#1] [General]
Hi!

Maybe an inappropriate way in doing this,
but how to get in contact to you Thi Garlet.

The possibility to get multi user on one gpu is very interesting to an project of us.

May you could send me an message, if you like to.

antonio.gottlieb@gmail.com

Best regards!

**Comment by garlett** [comment]
Hello, 

Share more about what you are doing

Ps. Not receiving notifications
